### DISCUSSION TITLE
Is it possible to change placeholder only for task item node?

### DISCUSSION THREAD

**Discussion by O4epegb** [ueberdosis/tiptap#3037] [General Questions]
I'm using [Placeholder extension](https://tiptap.dev/api/extensions/placeholder#placeholder), and works fine for the most types of the nodes, but I could not find a way to change placeholder for the [task item node](https://tiptap.dev/api/nodes/task-item). The node inside item node is actually a paragraph, so some way of determining that this paragraph is a descendant of the task item would be great, but I can find any way to do it.

Does anyone have any ideas how to deal with it?

**Comment by mrm2234** [comment]
Hey was this ever solved? I am also curious

**Reply by O4epegb** [reply]
Nope, I didn't. Switched to Slate/Plate in the end
